src/index.ts

```diff
@@ -551,6 +551,33 @@ class Envoy {
       return response_message;
     }, this.opts.NS);
 
+    // Temporary "state dump" logic for checking Holochain network peer consistency
+    this.ws_server.register("holo/tmp/state_dump", async ({ installed_app_id, cell_nick }) => {
```

Comment:
Why `/tmp/` are we going to be removing it?

Reply:
I assume so. We don't want anybody who can spin up a chaperone to be able to see the state dump of anybody else's holochain cells